packages/components/src/Avatar/src/Avatar.tsx

```diff
@@ -0,0 +1,243 @@
+import { BrokenImageRichIcon } from "@hopper-ui/icons";
+import { type ResponsiveProp, useStyledSystem, type StyledSystemProps, useResponsiveValue } from "@hopper-ui/styled-system";
+import { filterDOMProps } from "@react-aria/utils";
+import clsx from "clsx";
+import { type CSSProperties, forwardRef, type ForwardedRef, useMemo, type ReactNode, type ReactEventHandler } from "react";
+import { useContextProps } from "react-aria-components";
+
+import { Text, type TextProps } from "../../typography/Text/index.ts";
+import { type SizeAdapter, cssModule, type BaseComponentProps } from "../../utils/index.ts";
+
+import { AvatarContext } from "./AvatarContext.ts";
+import { RichIconAvatarImage } from "./RichIconAvatarImage.tsx";
+import { useImageFallback } from "./useImageFallback.ts";
+
+import styles from "./Avatar.module.css";
+
+export const GlobalAvatarCssSelector = "hop-Avatar";
+export type AvatarSize = "xs" | "sm" | "md" | "lg" | "xl" | "2xl";
+
+type AvatarImageBaseProps = React.HTMLProps<HTMLImageElement>;
```

Comment:
no `React.` import HTMLProps from react directly in the improt section. 

Also AvatarImageBaseProps is not necessary and should just be 

```    
imageProps?: HTMLProps<HTMLImageElement>;

```